Source/Core/Core/Core.cpp

```diff
@@ -814,7 +815,7 @@ void UpdateTitle()
 		SFPS = StringFromFormat("VI: %u - Input: %u - FPS: %.0f - VPS: %.0f - %.0f%%", (u32)Movie::g_currentFrame, (u32)Movie::g_currentInputCount, FPS, VPS, Speed);
 	else
 	{
-		SFPS = StringFromFormat("FPS: %.0f - VPS: %.0f - %.0f%%", FPS, VPS, Speed);
+		SFPS = StringFromFormat("FPS: %.0f - VPS: %.0f - %.0f%% - Skip: %d", FPS, VPS, Speed, SkippedFrames);
```

Comment:
Does this mean that the Skip: bit always will be displayed? That would be rather annoying, since most people never use frame skip.